linux/debian/8/foxpass_setup.py

```diff
@@ -127,9 +127,8 @@ def write_nslcd_conf(uris, basedn, binddn, bindpw, threads, idle_timelimit):
 # number of threads. one LDAP connction per thread.
 threads {threads}
 
-# Idle Timeout so we don't keep unused connections open foever
-# default ncsd is 600s, we'll do 24h to keep active sessions
-# available.
+# Set how long to keep ldap connections to foxpass open.
+# Default is 600s
```

Comment:
When I read "Default is 600s", I think that if I leave this parameter out, nslcd will default to 600s. In this case, it will default to infinity.